### PR TITLE
Added Authvia source for Review

### PR DESCRIPTION
**Pull request by pgaur-authvia**

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Authvia Source"
+---
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:3]:
```suggestion
---
title: Authvia Source
id: E5Y3BqhAg2
beta: true
---
```

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+TXT2PAY from [Authvia](https://www.authvia.com/) is the easiest way to engage your customers for bills, invoices, donations, disbursement and any other payment use case you may have. Get paid faster without the hassle of PCI compliance.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:5]:
```suggestion
TXT2PAY from [Authvia](https://www.authvia.com/){:target="_blank”} is the easiest way to engage your customers for bills, invoices, donations, disbursement and any other payment use cases you may have. Get paid faster without having to worry about PCI compliance.
```

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+This source is maintained by <Authvia>. For any issues with the source, [contact their Support team](mailto:support@authvia.com).
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:7]:
```suggestion
This source is maintained by Authvia. For any issues with the source, [contact the Authvia Support team](mailto:support@authvia.com).
```

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+2. Search for "Authvia" in the Sources Catalog, select Authvia, and click **Add Source**.
+3. On the next screen, give the Source a name configure any other settings.
+
+   - The name is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The name can be anything, but we recommend using something that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
+
+4. Click **Add Source** to save your settings.
+5. Copy the Write key from the Segment UI.
+6. Go to [https://www.authvia.com/forms/setup-twilio-segment-and-authvia/](https://www.authvia.com/forms/setup-twilio-segment-and-authvia/) and enter your Authvia Client ID and Secret, along with your Write Key. You will also pick your region and choose which events you would like to subscribe to. If you do not have an Authvia Client ID and Secret, request access here [(https://authvia.atlassian.net/servicedesk/customer/portal/1/group/1/create/31)](https://authvia.atlassian.net/servicedesk/customer/portal/1/group/1/create/31
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:19]:
```suggestion
1. From your workspace's [Sources catalog page,](https://app.segment.com/authvia/sources/catalog){:target="_blank”} click **Add Source**.
2. Search for "Authvia" in the Sources Catalog, select Authvia, and click **Add Source**.
3. On the next screen, give the Source a name configure any other settings.
   - The name is used as a label in the Segment app, and Segment creates a related schema name in your warehouse. The name can be anything, but we recommend using something that reflects the source itself and distinguishes amongst your environments (eg. Authvia_Prod, Authvia_Staging, Authvia_Dev).
4. Click **Add Source** to save your settings.
5. Copy the Write key from the Segment UI.
6. Go to [Connect your Segment account](https://www.authvia.com/forms/setup-twilio-segment-and-authvia/){:target="_blank”} and enter your Authvia Client ID, Secret, and Segment Write Key. You will also pick your region and choose which events you would like to subscribe to. If you do not have an Authvia Client ID and Secret, request access from [the Authvia Technical Support team](https://authvia.atlassian.net/servicedesk/customer/portal/1/group/1/create/31){:target="_blank”}.
```

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Stream
+
+Authvia uses our stream Source component to send Segment event data. It uses a server-side (select from `track`, `identify`) method(s) to send data to Segment. These events are then available in any destination that accepts server-side events, and available in a schema in your data warehouse, so you can query using SQL.
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:23]:
```suggestion
Authvia uses our stream Source component to send Segment event data. It uses a server-side (select from `track`, `identify`) method(s) to send data to Segment. These events are then available in any destination that accepts server-side events, and available in a schema in your data warehouse, so you can query using SQL.
```
```suggestion
Authvia uses the Authvia stream source component to send Segment event data. It uses a server-side Track or Identify method to send data to Segment. These events are then available in any destination that accepts server-side events and available in a schema in your data warehouse that you can query using SQL.
```

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------- |
+| Actions Created          | An action was created for a customer to complete. An Action could be a payment, signature, data collection and more.                  |
+| Actions Updated          | A customer completed the action. An Action could be a payment, signature, data collection and more.                                   |
+| Customer Created         | A new customer was created in Authvia                                                                                                 |
+| Customer Updated         | An existing customer was updated                                                                                                      |
+| Customer Deleted         | An existing Customer was Deleted from Authvia                                                                                         |
+| Business Process Created | A new Business Process (Conversation) was created for a customer                                                                      |
```
Comment (forstisabella) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:38]:
```suggestion
| Customer Created         | A new customer was created in Authvia.                                                                                                 |
| Customer Updated         | An existing customer was updated.                                                                                                      |
| Customer Deleted         | An existing Customer was Deleted from Authvia.                                                                                         |
| Business Process Created | A new Business Process (Conversation) was created for a customer.                                                                      |
```

**Comment by pgaur-authvia** [comment]
Update Changes as per the suggestion.

Please Review.

Thanks

**Review by forstisabella — APPROVED** [review]
lgtm! once @tcgilbert approves we'll be ready to merge

**Comment by tcgilbert** [comment]
@pgaur-authvia the docs look good, but we'll hold off on merging until the rest of the source review is completed. I sent some questions in the support ticket, but I will post them here as well: 

> Hi Will and Praveen,
>  
> Thanks for submitting the docs, as for reviewing your source, I have a couple questions:
>  
> Events
>  
> I took a look at [the source created in your workspace](https://app.segment.com/authvia/sources/authvia/debugger), but I'm not seeing any events populating in the debugger. I'll need to review some example event payloads, could you please send some data into this source, so I can review? 
>  
> Segment Region
>  
> Have you provided the option for users to select which region you are posting data to (US vs. EU endpoint)? If not, you will need to add this in order for your integration to be available in our EU catalog. More details on this here: https://segment.com/docs/partners/sources/#regional-segment

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+| Actions Created          | An action was created for a customer to complete. An Action could be a payment, signature, data collection and more.                  |
+| Actions Updated          | A customer completed the action. An Action could be a payment, signature, data collection and more.                                   |
+| Customer Created         | A new customer was created in Authvia.                                                                                                 |
+| Customer Updated         | An existing customer was updated.                                                                                                      |
+| Customer Deleted         | An existing Customer was Deleted from Authvia.                                                                                         |
+| Business Process Created | A new Business Process (Conversation) was created for a customer.                                                                      |
+| Business Process Updated | A customer has completed some, or all Actions in a business process. Alternatively, the Business Process was cancelled or it expired. |
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:39]:
```suggestion
| Business Process Updated | A customer has completed some, or all Actions in a business process. Alternatively, the Business Process was cancelled or it expired. |

## Event Properties

The table below list the properties included in the events listed above.

| Property Name   | Description               |
| --------------- | ------------------------- |
| `email_id`      | ID of the email           |
| `from_id`       | Sender email ID           |
| `email_subject` | Subject line of the email |
| `link`          | URL of the link clicked   |
```

Could you please add a table outlining the common properties users can expect across the events above?

**Review by tcgilbert — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/cloud-apps/authvia/index.md (hunk 7 above)
Comment (pgaur-authvia) [reply src/connections/sources/catalog/cloud-apps/authvia/index.md:39]: I have added the link to the Authvia Documentation with the Event Properties.

File: src/connections/sources/catalog/cloud-apps/authvia/index.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+## Event Properties
+
+Go to [Read Authvia Webhook Documentation](https://developer.authvia.com/v3.3/reference/create-webhook-subscription){:target="_blank”} and  read Authvia Documentation for the list of properties included in the events.
```
Comment (tcgilbert) [inline src/connections/sources/catalog/cloud-apps/authvia/index.md:43]:
```suggestion
## Event Properties

For a list of properties included in the event payloads, refer to [Authvia's Webhook Documentation](https://developer.authvia.com/v3.3/reference/create-webhook-subscription){:target="_blank”}.
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]

**Review by tcgilbert — APPROVED** [review]
lgtm. @forstisabella can we get this included in tomorrow's deploy?

**Review by forstisabella — APPROVED** [review]
yes! can do @tcgilbert

**Pull request merged**